components/modals/RefreshFloorPriceModal.js

```diff
@@ -1,7 +1,20 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
 import { motion } from 'framer-motion'
+import { changeRefreshFloorPriceStatus } from 'redux/reducers/walletSlice'
 
 const RefreshFloorPriceModal = () => {
+  const dispatch = useDispatch()
+  const { refreshFloorPriceStatus } = useSelector((state) => state.wallet)
+
+  useEffect(() => {
```

Comment:
Can this useEffect function, or the code within it, be exported out into a util? I see it's repeated in several places.

Reply:
Ohh rightt, I think I can make a reusable custom hook for this!